### PR TITLE
machines: add support for disk type in storage pool creation dialog

### PR DESCRIPTION
**Pull request by KKoukiou**
Relevant to: https://bugzilla.redhat.com/show_bug.cgi?id=1676594

### CODE REVIEW

**Comment by KKoukiou** [comment]
The css needs a bit fixing, adding needswork

**Comment by KKoukiou** [comment]
@garrett it looks like this now, how can I stretch the Source path input?
disk-pool

File: pkg/machines/components/storagePools/createStoragePoolDialog.jsx [hunk 1]
```diff
@@ -229,6 +237,44 @@ const StoragePoolSourceRow = ({ onValueChanged, dialogValues }) => {
                 <hr />
             </React.Fragment>
         );
+    else if (dialogValues.type == 'disk')
+        return (
+            <React.Fragment>
+                <label className='control-label' htmlFor='storage-pool-dialog-source'>
+                    {_("Source Path")}
+                </label>
+                <FormGroup
```
Comment (Gundersanne) [inline pkg/machines/components/storagePools/createStoragePoolDialog.jsx]: This really is nit-picky but the layout of `<Tag attribute=...` differes in 3 different ways across this fragment (One starting with a linebreak, one where the attributes are indented relative to the tagname, and one where attributes are indented relative the the first attribute).

Personally I prefer 
```
<Tag attribute1=
     attribute2=
     ...
```

If you'd prefer not to bother that's also fine :D

Comment (KKoukiou) [reply pkg/machines/components/storagePools/createStoragePoolDialog.jsx]: I will fix it, thanks :D

**Comment by KKoukiou** [comment]
For release notes;
disk-device-storage-pool

**Comment by KKoukiou** [comment]
Free of cockpit-machines failures (apart from the selenium/edge, which is failing in other places as well, and ALL test failed there, not only machines)

**Pull request merged**